backend/kesaseteli/docker-entrypoint.sh

```diff
@@ -35,7 +35,7 @@ fi
 if [[ ! -z "$@" ]]; then
     "$@"
 elif [[ "$DEV_SERVER" = "1" ]]; then
-    python ./manage.py runserver 0.0.0.0:8000
+    python ./manage.py runserver_plus 0.0.0.0:8000 --cert-file /tmp/cert.crt
```

Comment:
Also update README.md?

Reply:
Just noticed there's another PR for nginx ssl termination (https://helsinkisolutionoffice.atlassian.net/browse/YJDH-109)

Reply:
Yes. We're probably going to switch to that instead of this runserver_plus. The nginx setup still needs a bit of testing so we'll transfer to that when it is tested and proven to work well.